cms/djangoapps/contentstore/tasks.py

```diff
@@ -258,7 +258,7 @@ def export_olx(self, user_id, course_key_string, language):
         artifact.save()
     # catch all exceptions so we can record useful error messages
     except Exception as exception:  # pylint: disable=broad-except
-        LOGGER.exception(u'Error exporting course %s', courselike_key)
```

Comment:
Is the `LOGGER` is same as initiating logger with `logging.getLogger(__name__)`?